src/lib/components/forms/form/model/form-type-map.ts

```diff
@@ -60,9 +60,9 @@ export class FormTypeMap {
     };
     public readonly double: TerraFormTypeInterface = {
         inputMap: {
-            required: 'inputIsRequired'
+            required: 'isRequired'
```

Comment:
warum nimmt man dann nicht direkt als input `required` ?

Reply:
Sollten wir wenn auch bei allen FormComponents anpassen

Reply:
Theoretisch hast du Recht @Neill83. Wir könnten alle inputs so benennen wie wir möchten und dann in der Formtype-Map entsprechend mappen.
Einen Nachteil hätte das jedoch, und zwar falls jemand die alten Input namen in den options verwendet hat.. Das würde dann nicht mehr funktionieren.